notebooks/basic-usage.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>Probably "sampling stats" is a better description than "diagnostics"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>